ethexe/validator/Cargo.toml

```diff
@@ -20,6 +19,8 @@ log.workspace = true
 anyhow.workspace = true
 futures.workspace = true
 parity-scale-codec = { workspace = true, features = ["std", "derive"] }
-tempfile.workspace = true
 
 static_init = "1.0.3"
+
+#[dev-dependencies]
```

Comment:
Why is the section commented out, if `ethexe-db` is used only in tests?

Reply:
Done